presentation/read.go

```diff
@@ -11,21 +11,22 @@ import (
 	"archive/zip"
 	"fmt"
 	"io"
-	"io/ioutil"
+	"log"
 
 	"github.com/unidoc/unioffice"
+	"github.com/unidoc/unioffice/common/tempstorage"
 	"github.com/unidoc/unioffice/zippkg"
 )
 
 // Read reads a document from an io.Reader.
 func Read(r io.ReaderAt, size int64) (*Presentation, error) {
 	doc := newEmpty()
 
-	td, err := ioutil.TempDir("", "gooxml-pptx")
+	tmpPath, err := tempstorage.TempDir("unioffice-pptx")
 	if err != nil {
-		return nil, err
+		log.Fatalf("creating zip: %s", err)
```

Comment:
let's not panic here, can log the problem but should
return the error back up the stack so that it can be handled by the user

Reply:
fixed